src/pages/home/report/ReportActionsView.js

```diff
@@ -101,7 +101,10 @@ class ReportActionsView extends React.Component {
         });
 
         this.keyboardEvent = Keyboard.addListener('keyboardDidShow', () => {
-            if (!ReportActionComposeFocusManager.isFocused()) {
+            // When the keyboard shows up and then
+            // if not focused on the report action composer or the current scroll position is not at the bottom
+            // we won't scroll to the bottom here.
+            if (!ReportActionComposeFocusManager.isFocused() || this.currentScrollOffset !== 0) {
```

Comment:
Actually I'm sorry, I misread your proposal earlier and would like to change this. I think your proposal actually just stops scrolling to the bottom for any time the keyboard is opened, and that isn't what we want here. 

This actually needs to be more complicated, and I think I'll actually take this one over, sorry about that @TwizzyIndy